cmake/O2Utils.cmake

```diff
@@ -320,14 +321,55 @@ function(O2_GENERATE_EXECUTABLE)
       MODULE_LIBRARY_NAME ${PARSED_ARGS_MODULE_LIBRARY_NAME}
   )
 
-  ############### install the executable #################
-  install(TARGETS ${PARSED_ARGS_EXE_NAME} DESTINATION bin)
+  if (NOT ${PARSED_ARGS_INSTALL} OR ${PARSED_ARGS_INSTALL})
```

Comment:
@Barthelemy while trying to find the reason for https://github.com/AliceO2Group/AliceO2/issues/196 I stumbled over this line. Isn't that a tautology? What was your intent?